smarts/env/__init__.py

```diff
@@ -17,9 +17,27 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
+import subprocess
+from typing import List
+
 from gym.envs.registration import register
 
 register(
     id="hiway-v0",
     entry_point="smarts.env.hiway_env:HiWayEnv",
 )
+
+register(
+    id="intersection-v0",
+    entry_point="smarts.env.intersection_env:intersection_env",
+)
+
+
+def build_scenario(scenario: List[str]):
```

Comment:
nitpick but should `build_scenario` belong somewhere in `smarts/sstudio`? seems more related there

Reply:
Moved.